cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -454,6 +457,15 @@ public boolean verify(String s, SSLSession sslSession) {
                 args = args.subList(2,args.size());
                 continue;
             }
+            if(head.equals("-v")) {
```

Comment:
I agree it is not that important since it can be done via the j.u.l property file anyway